tests/batch.rs

```diff
@@ -1,92 +0,0 @@
-use galoy_cli::batch::Batch;
```

Comment:
why is the tests deleted?

Reply:
the tests you're referring to were associated with the previous iteration of batch payments, which are no longer relevant. 
Once this PR is approved and the implementation is finalized, I'll be introducing new e2e tests for batch in a subsequent PR

Reply:
> Once this PR is approved and the implementation is finalized, I'll be introducing new e2e tests for batch in a subsequent PR

imo, unless exception for good reasons, tests should come with PRs, not after. otherwise it's too easy to move on to other things and never do the tests.